{{cookiecutter.repo_name}}/production.ini

```diff
@@ -20,6 +20,12 @@ retry.attempts = 3
 # wsgi server configuration
 ###
 
+[alembic]
+# path to migration scripts
+script_location = {{ cookiecutter.repo_name }}/alembic
+file_template = %%(year)d_%%(month).2d_%%(day).2d_%%(hour).2d_%%(minute).2d_%%(second).2d_%%(rev)s
```

Comment:
I'm not sure we should change the default file_template versus stock alembic

Reply:
I vote for the date based file template, otherwise it's super hard to find the latest revision.

Reply:
@mmerickel what are you unsure about? Can you elaborate?

@hyperknot does Alembic then support using the custom identifiers as [partial identifiers](http://alembic.zzzcomputing.com/en/latest/tutorial.html#partial-revision-identifiers), e.g., `alembic upgrade 20180608141155`?

Would we lose any feature from Alembic with custom identifiers?

Reply:
I'd propose the following file template if we're going to change it.

```
file_template = %%(year)d%%(month).2d%%(day).2d_%%(rev)s
```

It's simple and sortable without too much information in it. Compare `20180611_25c1450def88.py` to your proposed `2018_06_11_12_02_37_25c1450def88.py`.

Reply:
I like that because there is a single underscore that allows me to easily parse the revision for `alembic upgrade 25c`. If I need further sorting for a given date, I can look at file timestamps.

Reply:
Timestamps disappear in git, but overall I guess the shorter version is a good compromise.